docs/user-guide/global-configurations/deployment-window.md

```diff
@@ -0,0 +1,279 @@
+
+#### Bulk Apply
+
+1. If you wish to apply deployment windows to multiple applications at once, use the checkbox to select the applications. 
+
+    ![Figure 13: Selecting Apps](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/select-apps.jpg)
+
+    We recommend you to use the available filters (Application, Environment, Deployment Window) to simplify the process of application selection.
+
+    ![Figure 14a: Using Filters](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/filter-apply.jpg)
+
+    ![Figure 14b: Filtered Results](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/filter-apply-2.jpg)
+
+2. On the floating widget, click **Manage Windows**.
+
+    ![Figure 15: Manage Windows](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/manage-window.jpg)
+
+3. Click the **Add Deployment Windows** dropdown to choose the deployment window(s).
+
+    ![Figure 16: Attaching Windows to Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/attach-windows.gif)
+
+4. Use the **Review Changes** option to confirm the impacted environment(s) and click **Save**.
+
+    ![Figure 17: Checking Impacted Environments](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/review-environment.gif)
+
+You can remove deployment window(s) applied to one or more apps as shown below.
+
+![Figure 18a: Removing Windows from Single App](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/remove-window-single.gif)
+
+![Figure 18b: Removing Windows from Multiple Apps](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/remove-window.gif)
+
+
+---
+
+## Checking Deployment Window
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have view-only permission or above (along with access to the environment and application) to check deployment windows.
+{% endhint %}
+
+### Overview Page
+
+The **Deployment window** section shows the deployment windows configured for each [environment](../../reference/glossary.md#environment).
+
+![Figure 19: Overview Page - Deployment Window](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/deployment-window.jpg)
+
+However, if a deployment window doesn't exist for an environment, the message `No deployment windows are configured` would be displayed next to it.
+
+You may click the dropdown icon to view the details which include:
+* Type of deployment window (Blackout/Maintenance)
+* Name and description
+* Frequency of window (once, weekly, monthly, yearly)
+* Start date and time 
```

Comment:
Need to confirm if this is only Start/End date OR Start/End date and time.

Reply:
Actually, the date is never shown on Overview page it seems. I think it was an oversight from my end.
Now I have replaced them with 'duration' to avoid confusion since start date and end date are integral part of 'Add duration' functionality.